types/woosmap.map/tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "ban-types": false,
+        "max-line-length": false,
+        "no-mergeable-namespace": false,
+        "no-unnecessary-class": false,
+        "no-unnecessary-qualifier": false
```

Comment:
Why is `no-unnecessary-class` failing?
It's difficult with generated code, but ideally no lint rules would have to be disabled.

Reply:
Thanks for your feedback!
We fixed the generated type defs to be able to remove additional lint rules.
The rules `no-unnecessary-qualifier` will be hard to remove as some interfaces have common names but under different sub namespaces. Same for `no-mergeable-namespace` which will be hard to fix in our generation tool.